public/scripts/statusDisplay.js

```diff
@@ -0,0 +1,35 @@
+const displayDiv = document.querySelector(".display-div");
+
+export const updateStatusDisplay = (state) => {
+  let displayStatusMessage = displayDiv.innerText;
+  if (state.gameOverState.isOver) {
+    displayStatusMessage = generateWinnerDisplayMessage(state.gameOverState);
+  } else {
+    displayStatusMessage = generateTurnDisplay(state.players)
+  }
+
+  displayDiv.innerText = displayStatusMessage;
+}
+
+const generateWinnerDisplayMessage = (gameOverState) => {
+  const winner = gameOverState.winner;
+  return (winner === null) ? "It's a tie!" : winner + " wins!";
```

Comment:
Let's use string interpolation in all the places where you are concatenating strings